### ISSUE TITLE
[Feature]: Improve accessibility on rte

### ISSUE THREAD

**Issue by nac62116** [mint-vernetzt/community-platform#1056]
### What do you like to be improved?

Some awesome stuff!

**Comment by nac62116** [comment]
More than only the rte field. The whole settings general form cannot be submitted without javascript. We should work on this to improve accessibility.
